web_src/less/shared/issuelist.less

```diff
@@ -23,13 +23,29 @@
     .issue-item-main {
       width: 100%;
     }
+    
+    .issue-item-main.action-issue-item {
+      width: 80%;
+    }
```

Comment:
Does the element need a separate CSS class name, instead of overwriting existing `issue-item-main`? It really looks strange.

Reply:
> Does the element need a separate CSS class name, instead of overwriting existing `issue-item-main`? It really looks strange.

I gave it the new name "action-issue-item", this selector is used to match only this issue-item-main, i can remove `.issue-item-main` here since `action-issue-item` is only used here.